bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -196,7 +202,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         return;
                     }
                 }
+                break;
             default:
+                if (command instanceof RefreshType) {
+                    ScheduledFuture<?> job = pollingJob;
+                    if (job != null) {
+                        job.cancel(true);
+                    }
+                    pollingJob = scheduler.scheduleWithFixedDelay(this::getStatus, 0, config.refreshInterval,
+                            TimeUnit.MINUTES);
+                }
```

Comment:
Thanks for this suggestion.  It works (I think) and I have removed the update now switch channel.  I want my user to test this and maybe we will add some documentation if it is OK from a code perspective.  I have added null pointer handling as per the warning.